<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="2" hostname="DESKTOP-VG7FOF4" name="com.QaLegendBilling.Scripts.ProductsExpenseTest" tests="3" failures="0" timestamp="2023-03-20T12:02:39 IST" time="0.354" errors="1">
  <testcase name="verifyLoginValidUserValidPassword" time="0.354" classname="com.QaLegendBilling.Scripts.ProductsExpenseTest">
    <error type="org.apache.poi.openxml4j.exceptions.NotOfficeXmlFileException" message="No valid entries or contents found, this is not a valid OOXML (Office Open XML) file">
      <![CDATA[org.apache.poi.openxml4j.exceptions.NotOfficeXmlFileException: No valid entries or contents found, this is not a valid OOXML (Office Open XML) file
at org.apache.poi.openxml4j.util.ZipArchiveThresholdInputStream.getNextEntry(ZipArchiveThresholdInputStream.java:156)
at org.apache.poi.openxml4j.util.ZipInputStreamZipEntrySource.<init>(ZipInputStreamZipEntrySource.java:94)
at org.apache.poi.openxml4j.opc.ZipPackage.<init>(ZipPackage.java:132)
at org.apache.poi.openxml4j.opc.OPCPackage.open(OPCPackage.java:312)
at org.apache.poi.ooxml.util.PackageHelper.open(PackageHelper.java:59)
at org.apache.poi.xssf.usermodel.XSSFWorkbook.<init>(XSSFWorkbook.java:304)
at org.apache.poi.xssf.usermodel.XSSFWorkbook.<init>(XSSFWorkbook.java:300)
at com.QaLegendBilling.Utilities.ExcelUtilities.getCellStringData(ExcelUtilities.java:19)
at com.QaLegendBilling.Scripts.LoginTest.verifyLoginValidUserValidPassword(LoginTest.java:64)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
Caused by: java.util.zip.ZipException: Unexpected record signature: 0X8858088E
at org.apache.commons.compress.archivers.zip.ZipArchiveInputStream.getNextZipEntry(ZipArchiveInputStream.java:296)
at org.apache.poi.openxml4j.util.ZipArchiveThresholdInputStream.getNextEntry(ZipArchiveThresholdInputStream.java:152)
... 23 more
]]>
    </error>
  </testcase> <!-- verifyLoginValidUserValidPassword -->
  <system-out/>
  <testcase name="addUnits" time="0.000" classname="com.QaLegendBilling.Scripts.ProductsExpenseTest">
    <skipped/>
  </testcase> <!-- addUnits -->
  <system-out/>
  <testcase name="addExpenses" time="0.000" classname="com.QaLegendBilling.Scripts.ProductsExpenseTest">
    <skipped/>
  </testcase> <!-- addExpenses -->
  <system-out/>
</testsuite> <!-- com.QaLegendBilling.Scripts.ProductsExpenseTest -->
